_posts/2019-14-11-designup-bangalore.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: event
+title:  "Open Design @ DesignUP!: Bangalore, Nov 14th"
+date:   2019-14-11
```

Comment:
```suggestion
date:   2019-11-14
```
I think you should be able to accept this change suggestion, then it will simply fix it. :)

Reply:
Nevermind, merged your other pull request in here. :)